std/socket.d

```diff
@@ -2808,13 +2808,26 @@ public:
     /**
      * Accept an incoming connection. If the socket is blocking, $(D accept)
      * waits for a connection request. Throws $(D SocketAcceptException) if
-     * unable to _accept. See $(D accepting) for use with derived classes.
+     * unable to accept. See $(D accepting) for use with derived classes.
      */
-    Socket accept() @trusted
+    Socket accept() @safe
+    {
+        auto newSocket = tryAccept();
+        if (newSocket is null)
+            throw new SocketAcceptException("Unable to accept socket connection");
+        return newSocket;
+    }
+
+    /**
+     * Accept an incoming connection. If the socket is blocking, $(D tryAccept)
+     * waits for a connection request. Returns $(D null) if unable to accept.
```

Comment:
Please use "Returns:" (i.e. e Ddoc format) and state that a `Socket` object is returned by default (I know it's obvious, but we're trying to make the docs better and more intuitive).